src/main/java/com/brainpix/profile/controller/PortfolioController.java

```diff
@@ -0,0 +1,85 @@
+package com.brainpix.profile.controller;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.brainpix.CommonPageResponse;
+import com.brainpix.api.ApiResponse;
+import com.brainpix.profile.dto.request.PortfolioRequest;
+import com.brainpix.profile.dto.response.PortfolioDetailResponse;
+import com.brainpix.profile.dto.response.PortfolioResponse;
+import com.brainpix.profile.service.PortfolioService;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/portfolios")
+@RequiredArgsConstructor
+public class PortfolioController {
+
+	private final PortfolioService portfolioService;
+
+	@PostMapping
+	public ResponseEntity<ApiResponse<Long>> createPortfolio(
+		@RequestParam long userId,
+		@RequestBody PortfolioRequest request
+	) {
+		try {
+			Long portfolioId = portfolioService.createPortfolio(userId, request);
+			return ResponseEntity.ok(ApiResponse.success(portfolioId));
```

Comment:
저는 201 Created로 응답하는 편인데 이부분은 동준님 편하신대로 해도 될거같습니다! 프런트분들이랑도 맞춰봐야 할것 같아요
```suggestion
			return new ResponseEntity<>(data, HttpStatus.CREATED);
```